.circleci/config/commands/go_test.yml

```diff
@@ -58,21 +58,38 @@ steps:
         USE_DOCKER=1
         <</ parameters.use_docker >>
 
-        # Split Go tests by prior test times.  If use_docker is true, only run
-        # tests that depend on docker, otherwise only those that don't.
-        if [ $USE_DOCKER == 1 ]; then
-          package_names=$(go list -test -json ./... |
-            jq -r 'select(.Deps != null) |
-              select(any(.Deps[] ; contains("github.com/hashicorp/vault/helper/testhelpers/docker"))) |
-              .ForTest | select(. != null)' |
-              sort -u | grep -v vault/integ | circleci tests split --split-by=timings --timings-type=classname)
-        else
-          package_names=$(go list -test -json ./... |
-            jq -r 'select(.Deps != null) |
-              select(all(.Deps[] ; contains("github.com/hashicorp/vault/helper/testhelpers/docker")|not)) |
-              .ForTest | select(. != null)' |
-              sort -u | grep -v vault/integ | circleci tests split --split-by=timings --timings-type=classname)
-        fi
+        # Check all directories with a go.mod file
+        modules=('.' 'api' 'sdk')
+        all_package_names=""
+
+        for dir in "${modules[@]}"
+        do
+          cd $dir
+          # Split Go tests by prior test times.  If use_docker is true, only run
```

Comment:
nit: can we remove the newline below this line?

Reply:
hmm, there isn't a newline below this, I think that might be github's ui being weird ?